gallery/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "1.5.0",
+  "version": "1.6.0",
```

Comment:
Why isn't this a patch version?

Reply:
> Why isn't this a patch version?

Changed to 1.5.1 https://github.com/sonatype/sonatype-react-shared-components/pull/129/commits/e3c6a14749029e9cbb246b38cc66cd4dc7994451